<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDiagramViewControlPanel</class>
 <widget class="QWidget" name="QDiagramViewControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QDiagramViewControlPanel</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="snapToolButton">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enables or disables the view's 'snap to grid' mode&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="qdiagramlib.qrc">
       <normaloff>:/qdiagram/snap.disabled</normaloff>:/qdiagram/snap.disabled</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QToolButton" name="showGridToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="qdiagramlib.qrc">
         <normaloff>:/qdiagram/grid.visible</normaloff>:/qdiagram/grid.visible</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="hideGridToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="qdiagramlib.qrc">
         <normaloff>:/qdiagram/grid.invisible</normaloff>:/qdiagram/grid.invisible</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="mousePositionLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>0,0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDiagramViewZoomSlider" name="zoomSlider"/>
   </item>
  </layout>
  <zorder>zoomSlider</zorder>
  <zorder>mousePositionLabel</zorder>
  <zorder>snapToolButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDiagramViewZoomSlider</class>
   <extends>QWidget</extends>
   <header>qdiagramviewzoomslider.h</header>
   <container>1</container>
   <slots>
    <signal>zoomChanged(int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qdiagramlib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>snapToolButton</sender>
   <signal>clicked()</signal>
   <receiver>QDiagramViewControlPanel</receiver>
   <slot>snapToolButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>11</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>snapToolButtonClicked()</slot>
 </slots>
</ui>
